packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -154,17 +155,31 @@ export class AccountResolver {
     return !!result;
   }
 
+  @Features(AdminPanelFeature.EnableAccount)
+  @Mutation((returns) => Boolean)
+  public async enableAccount(
```

Comment:
Can we get this under test? I noticed that disableAccount also isn't under test. If possible adding cursory tests for both would good.

Reply:
Definitely, great idea